src/styles/style.scss

```diff
@@ -0,0 +1,127 @@
+@import './utils/variables';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.product {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: $white-accent;
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  &_img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &_name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    width: $width-content;
+    color: $main-text-color;
+  }
+
+  &_code {
+    font-size: 10px;
+    line-height: 14px;
+    color:$secondary-text-color;
+    margin-top: 4px;
+    margin-bottom: 16px;
+  }
+}
+
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  width: $width-content;
+  height: 16px;
+
+  &_amount {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    color: $main-text-color;
+
+    text-align: right;
+    font-size: 10px;
+    line-height: 14px;
+  }
+}
+
+.stars {
```

Comment:
you must have here only two blocks: `product`, `stars`. Move each block to separate file